### ISSUE TITLE
running speed is too slow, only using one CPU

### ISSUE THREAD

**Issue by billzt** [OpenGene/gencore#7]
the reference genome has 26 chrs, 2.4G size. And the sequence coverage is about 100X.

Currently gencore keeps running for >15 hours and still doesn't stop.

I found that gencore uses only one CPU all the time, and I think whether it would be better to change to multi-workers in the future.

**Comment by zhujiaqi2014** [comment]
这个可能不是cpu问题，我得数据量是你的100倍，40分钟搞完。估计gencore不适配你得数据，或者是用法没对

**Comment by zhujiaqi2014** [comment]
This may not be a cpu problem. I have 100 times more data than you, and it takes 40 minutes. It is estimated that gencore does not fit your data, or the usage is incorrect.

Read the doc may be helpful

**Comment by zhujiaqi2014** [comment]
Oh, I'm wrong, not 100 times, but almost the same with your data. 

Actually, this gencore may not be suitable for your data. In order to detect low frequency mutations, there are many steps to correct mismatch bases, and these steps not be useful for your data. you may want to find another software to markdup, such as picard

**Comment by billzt** [comment]
@zhujiaqi2014 Well, do you mean that gencore is only suitable to deal with somatic mutations in cancer genomics rather than other situations?

If so, it should be noted in the README.

**Comment by zhujiaqi2014** [comment]
@billzt Yes, I think it is made for somatic mutation, especially the low frequency ones. This software has many steps to correct the basecalling which is important to low maf mutations. Actually, it is doing quite well in my “umi” data.
But if you just want to dedup bam and call Germaine mutations. You may want to use the speedseq pipeline.

**Comment by antonkulaga** [comment]
@zhujiaqi2014 I am actually using it for germline calling and runtime seems good for me. Do you have anything against germline usecase?
